File: 2238/2238.cs
public class Solution {
    public int NumberOfTimesADriverWasAPassen(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(NumberOfTimesADriverWasAPassen(root.left), NumberOfTimesADriverWasAPassen(root.right));
    }
}

File: 2238/2238.go
func numberOfTimesADriverWasAPassen(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := numberOfTimesADriverWasAPassen(root.Left)
	r := numberOfTimesADriverWasAPassen(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}